static-assets/components/cstudio-forms/template-editor.js

```diff
@@ -108,6 +108,24 @@ CStudioAuthoring.Module.requireModule(
                 }
 
                 me.renderTemplateEditor(templatePath, content, onSaveCb, contentType, mode);
+              }).catch(error => {
+                const errorMsg = error.responseText
```

Comment:
Is `CStudioAuthoring.Service.getContent` API2 too?

(Might you get different error objects from the two promises?)

Reply:
Yes, that is handled, if you see line 112 it verifies the error property, depending of it sets the error message one way or another